Dockerfile

```diff
@@ -1,21 +1,13 @@
-FROM openjdk:17-slim
+FROM arm64v8/eclipse-temurin:17
 
-MAINTAINER Benjamin Rögner "benjamin.roegner@here.com"
-MAINTAINER Lucas Ceni "lucas.ceni@here.com"
-MAINTAINER Dimitar Goshev "dimitar.goshev@here.com"
+ARG NAKSHA_VER
+RUN test -n ${NAKSHA_VER:?}
 
-ENV LOG_CONFIG log4j2-console-plain.json
-ENV LOG_PATH /var/log/xyz
-ENV FS_WEB_ROOT www
+RUN mkdir /opt/app
+ENV NAKSHA_CONTAINER_JAR /opt/naksha-${NAKSHA_VER}-all.jar
+COPY build/libs/naksha-${NAKSHA_VER}-all.jar $NAKSHA_CONTAINER_JAR
 
-#Override the following environment variables to let the service connect to different host names
-ENV STORAGE_DB_URL jdbc:postgresql://postgres/postgres
-ENV PSQL_HOST postgres
-ENV XYZ_HUB_REDIS_URI redis://redis
-ENV PSQL_HTTP_CONNECTOR_HOST xyz-http-connector
+ENV CONFIG_ID test-config
```

Comment:
let's call the environment variables as NAKSHA_CONFIG_ID and NAKSHA_ADMIN_DB_URL (L instead of I - to match with what we are already using during deployment today)